### ISSUE TITLE
Character creation - Traits tab - UI audit

### ISSUE THREAD

**Issue by ZeroInternalReflection** [CleverRaven/Cataclysm-DDA#58265]
### Describe the bug

The traits tab of the character creation screen has several graphical and navigational issues.  Some of these overlap with other character creation issues, but most appear to be independent.  These include:

**Display issues**:
1] Appearance names are cut off on small screens
2] Trait costs wrap around and print over the border on small screens
![Character creation - Traits - Cut off text – 80x24](https://user-images.githubusercontent.com/89038572/168186425-f66bc8ee-dd71-4539-a4e2-a20d62243541.png)
3] Trait names are cut off on small screens in non-English languages (Polish shown):
![Character creation - Traits --  80x24 – Polish](https://user-images.githubusercontent.com/89038572/168186424-56cc8043-181a-4ddb-b6de-13524ad4a8bd.png)
4] Some trait descriptions are cut off in non-English languages
5] In Russian, and possibly other non-English languages, changing the sort mode changes the location of the sort hint
![Character creation - Traits - Sorted by points -- 80x24 -- Russian - With cut off description](https://user-images.githubusercontent.com/89038572/172513937-b2cc8b96-71ea-4c2b-bf1d-6b3bbd923100.png)
![Character creation - Traits - Unsorted -- 80x24 -- Russian](https://user-images.githubusercontent.com/89038572/172513943-3cdc6b23-481b-465a-8293-862ac10e0ca3.png)
**Accessibility issues**:
6] The cursor is not set for screen readers
**Navigation issues**:
7] No mouse support is available
8] A hint is shown for filtering, but the filter keybinding is not shown

### Steps to reproduce

1. Open the game
2. Set language specified (if applicable)
3. Resize window to size specified (if described as 'small', assume 80x24)
4. New game
5. Custom character
6. `Tab` to "TRAITS"

### Expected behavior

- [ ] 1] Appearance names should truncate/fold cleanly at all screen sizes in all supported languages
- [ ] 2] Change to "This trait costs x points"
- [ ] 3] Trait names should truncate/fold cleanly
- [ ] 4] Trait descriptions should be scrollable
- [ ] 5] The keybinding hints should be in a consistent location
- [ ] 6] The cursor should be set for screen readers according to [accessibility.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/ACCESSIBILITY.md)
- [ ] 7] It should be possible to: toggle traits with left-click, scroll through active trait list with mousewheel, and select a different tab with left click
- [ ] 8] The filter hotkey `/` should be shown in the filter hint

### Versions and configuration

<details><summary>Tiles</summary>

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: Arch; Description: Arch Linux; Release: rolling; Codename: n/a; 
- Game Version: 90e1eb2 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]
</details>
<details><summary>Curses</summary>

### Additional context

For clarification of some of my UI terminology (e.g. scroll vs. fast scroll, column vs. pane), refer to my general notes [here](https://gist.github.com/ZeroInternalReflection/a4a1bbd30cf5fbee4141b5f0e7d6719a)

**Comment by Zireael07** [comment]
Hey OP, maybe it's worth creating a tracker for those? I think it's like the 5th detailed audit that you've submitted (Awesome btw!)

**Comment by NetSysFire** [comment]
>maybe it's worth creating a tracker for those?

You mean a project? This kanban-thing at https://github.com/CleverRaven/Cataclysm-DDA/projects?
Unfortunately it requires certain special permissions to the repo to modify so adding and removing items is impossible to do on your own

**Comment by Zireael07** [comment]
No, a tracker issue. An issue that is formatted as a to-do list with a list of links to all the related issues.

**Comment by ZeroInternalReflection** [comment]
Some sort of central tracking for these issues (and other related UI issues) is probably a good idea.  There was a brief discussion of setting up a project for this on Discord.  Of course, there are already a couple of (largely dormant, I think) UI projects there.
I'm not sure whether a project or a tracking issue would be the better way to track these.
